tests/test_main.py

```diff
@@ -4,122 +1,20 @@
+# tests/test_main.py
+import unittest
+from car_wash_station import CarWashStation  # Импортируем модуль из корня проекта
 
-from app.main import Car, CarWashStation
+class TestCarWashStation(unittest.TestCase):
+    def setUp(self):
+        self.station = CarWashStation()
 
+    def test_add_car(self):
+        self.station.add_car('Car 1')
```

Comment:
The `add_car` method is not defined in the `CarWashStation` class. You need to implement this method or adjust the test to match the available methods.